src/com/lightdev/app/shtm/resources/SimplyHTML_de.properties

```diff
@@ -74,7 +74,6 @@ pasteLabel=Einf\u00FCgen
 pasteTip=Einf\u00FCgen
 pastePlainTextLabel=Einf\u00FCgen als Text
 pasteHTMLLabel=Einf\u00FCgen als HTML
-pastePlainTextTip=Einf\u00FCgen als Text (ohne Formatierung)
```

Comment:
Why have you removed it?

Reply:
it is commented out in default (English) properties file which I also have removed. There is no references to this property in the code